src/Client.ts

```diff
@@ -222,6 +230,13 @@ export function defaultOptions(opts?: Partial<ClientOptions>): ClientOptions {
     opts.codecs = _defaultOptions.codecs.concat(opts.codecs)
   }
 
+  if (opts?.useSnaps) {
+    opts.keystoreProviders = [
+      new SnapProvider(`npm:${snapPackage}`, snapVersion),
```

Comment:
Noob question, what do we do with this version eventually?

Reply:
I'm putting together a bigger doc around it, but we use it to upgrade the version of the user's installed Snap if the specified version is higher than their installed version.